buildenv/jenkins/common/variables-functions

```diff
@@ -360,20 +360,6 @@ def set_node(job_type) {
     }
 }
 
-/*
-* Set the RELEASE variable with the value provided in the variables file.
-*/
-def set_release() {
-    RELEASE = get_value(VARIABLES."${SPEC}".release, SDK_VERSION)
-}
-
-/*
-* Set the JDK_FOLDER variable with the value provided in the variables file.
-*/
-def set_jdk_folder() {
```

Comment:
`set_jdk_folder()` is still required by the build() method - see `stage('Java Version'){...}` in common/build script

Reply:
Good catch. Odd I thought I had done a PR build before but it clearly fails now. Going to try `*` in the path and see if that works.

Reply:
That won't work. Added these functions back in.